pimd/pim_cmd.c

```diff
@@ -3162,9 +3166,11 @@ static void pim_show_group_rp_mappings_info(struct pim_instance *pim,
 
 		frr_each (bsm_rpinfos, bsgrp->bsrp_list, bsm_rp) {
 			char rp_str[INET_ADDRSTRLEN];
+			struct prefix bsm_rp_addr;
 
-			pim_inet4_dump("<Rp Address?>", bsm_rp->rp_address,
-				       rp_str, sizeof(rp_str));
+			pim_addr_to_prefix(&bsm_rp_addr, bsm_rp->rp_address);
```

Comment:
Why convert to `struct prefix` here? Just print `bsm_rp->rp_address` with `%pPA`?

BTW: for JSON you can use `json_object_string_addf(…, "%pPA", &bsm_rp->rp_address);`

Reply:
Fixed